src/components/pages/HomePage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Navigate, useParams } from 'react-router-dom';
+import { NotFoundPage } from './NotFoundPage';
+
+export const HomePage = () => {
+  const { home } = useParams();
+
+  if (home === 'home') {
+    return <Navigate to="/" replace={true} />;
+  } else if (home) {
+    return <NotFoundPage />;
+  }
```

Comment:
The conditional logic here is a bit confusing. It seems like you're trying to redirect to the root URL if `home` equals 'home', and show a not found page if `home` has any other truthy value. This seems unusual for a home page. A home page typically wouldn't have this kind of logic, it would just render the home page content. It might be more appropriate to move this logic to a router or another component that handles routing.